packages/mp3-encoder/CHANGELOG.md

```
@@ -0,0 +1,143 @@
+## 0.31.26
+
+### Patch Changes
+
+- [#1481](https://github.com/RocketChat/fuselage/pull/1481) [`8084488`](https://github.com/RocketChat/fuselage/commit/80844881787fb8c5679ac5bcd18e43ddc23a8673) Thanks [@tassoevan](https://github.com/tassoevan)! - Suppressed assertions at the LAME encoding due to unknown `NaN` values
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+# [0.31.0](https://github.com/RocketChat/fuselage/compare/v0.30.1...v0.31.0) (2021-12-28)
+
+### Features
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)